tests/jest/aggregate-translations.test.js

```diff
@@ -49,6 +49,15 @@ describe('aggregate-translations', () => {
     expect(searchedDirectories).toEqual(expect.arrayContaining([`${process.cwd()}${path.sep}test${path.sep}*${path.sep}pattern`]));
   });
 
+  it('aggregates on the default search patterns and custom directory patterns while excluding the custom excludes directory patterns', () => {
+    aggregateTranslations({ directories: ['./test/*/pattern', './foo/*/bar', './baz/*/buzz'], excludes: ['./foo/*/bar'] });
+
+    expect(globSpy).toHaveBeenCalledTimes(6);
+    expect(searchedDirectories).toEqual(expect.arrayContaining([`${process.cwd()}${path.sep}test${path.sep}*${path.sep}pattern`]));
```

Comment:
This can be simplified to be `expect(searchedDirectories).toEqual(expect.arrayContaining([expect.stringContaining(`${path.sep}test${path.sep}*${path.sep}pattern`]));

Reply:
Lets make sure we call out that exclude will be honored before a search pattern / directory pattern included. 👍